README.md

```diff
@@ -10,3 +11,17 @@ uvicorn qr-generator:app --reload
 ```
 
 - Ensure [public](https://docs.github.com/en/codespaces/managing-codespaces-for-your-organization/restricting-the-visibility-of-forwarded-ports#overview) port forwarding in codespace.
+
+## Running the Dockerfile
+
+```bash
+$ docker build -t my-fastapi-app .
```

Comment:
Rather than calling it fastapi-app I would probably do qr or something else that represents what you have built. Ex. qr-code

Reply:
agree